.stylelintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "rules": {
+      "block-no-empty": null,
+      "color-no-invalid-hex": true,
+      "comment-empty-line-before": [ "always", {
+        "ignore": ["stylelint-commands", "after-comment"]
+      } ],
+      "declaration-colon-space-after": "always",
+      "indentation": [2, {
+        "except": ["value"]
+      }],
+      "max-empty-lines": 4,
+      "rule-empty-line-before": [ "always", {
+        "except": ["first-nested"],
+        "ignore": ["after-comment"]
+      } ],
+      "unit-whitelist": ["em", "rem", "%", "s", "px", "vh"]
+    }
+  }
```

Comment:
Add an empty line at the end of the file.  (You can set your editor to do this for you)